frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -167,7 +167,9 @@
   "Change Subscription update channel": "Change Subscription update channel",
   "Which channel is used to receive updates?": "Which channel is used to receive updates?",
   "Uninstall Operator?": "Uninstall Operator?",
-  "This will remove Operator <1>{{name}}</1> from <4>{{namespace}}</4>. Removing the Operator will not remove any of its custom resource definitions or managed resources. If your Operator has deployed applications on the cluster or configured off-cluster resources, these will continue to run and need to be cleaned up manually.": "This will remove Operator <1>{{name}}</1> from <4>{{namespace}}</4>. Removing the Operator will not remove any of its custom resource definitions or managed resources. If your Operator has deployed applications on the cluster or configured off-cluster resources, these will continue to run and need to be cleaned up manually.",
+  "Operator <1>{{name}}</1> will be removed from <4>{{namespace}}</4>, but any of its custom resource definitions or managed resources will remain. If your Operator deployed applications on the cluster or configured off-cluster resources, these will continue to run and require manual cleanup.": "Operator <1>{{name}}</1> will be removed from <4>{{namespace}}</4>, but any of its custom resource definitions or managed resources will remain. If your Operator deployed applications on the cluster or configured off-cluster resources, these will continue to run and require manual cleanup.",
```

Comment:
Do we need to make it more clear that this is being removed from a namespace? 
Based on the PR it seems like we are setup to convey this to the user as best we can; but it unclear how clear this messaging is.

Reply:
We do call out "Operator" before we list the name of the Operator so we could probably do the same for "Project" if you think that level of information would make it more clear.